### DISCUSSION TITLE
How do I position images in Markdown?

### DISCUSSION THREAD

**Discussion by uranomaloka** [datenstrom/community#665] [Ask a question] [answered]
I am trying to align an Image on the right Side with [image example.jpg right - 30%], but it shows no difference to [image Bild1.jpg left - 30%] or [image Bild1.jpg center - 30%]. It's the same problem if I remove the size tag (- 30%).
Does anyone have a solution to this or am I missing something?

**Comment by schulle4u** [comment]
It looks your image shortcut arguments are in wrong order. Please have a look at some [examples in the extension documentation](https://github.com/datenstrom/yellow-extensions/tree/master/source/image) for the right syntax.

**Reply by uranomaloka** [reply]
Thank you for answering. I have already tried some different orders. Looking at the Image Extension Website, I sadly still can't seem to find what I am doing wrong here.

**Reply by IngoBelka** [reply]
Could it be, that your used theme/css doesn't support this? I'm currently not sure if all themes have the alignment included.

**Reply by uranomaloka** [reply]
I have indeed used a custom Theme, but trying it with the default stockholm Theme makes no difference.

**Comment by nzyeehaa** [comment, marked as answer]
@uranomaloka - from your examples,
[image example.jpg right - 30%] {a picture called example.jpg, and it is named "right" with no alignment and 30% original size}
[image Bild1.jpg left - 30%]{a picture called Bild1.jpg, and it is named "left" with no alignment and 30% original size}
[image Bild1.jpg center - 30%] {a picture called Bild1.jpg, and it is named "center", with no alignment and 30% original size}

explanation...
assuming you have image extension installed...
[image {file} {alt txt} {align} {size}]
if you dont require one setting - replace it with a dash if something follows
[image pic.jpg - {align} {size}]
[image pic.jpg - right]

Suggestions...
**first** - be careful with camelcase or capitals for images - I find it is better for images to be saved/named in lower case for some servers.
**second** - if you are using a standard template try this [image bild1.jpg "this is the alt txt of the image" right 30%]
**third** - the general templates provided are perfect for general use, if you should look to custom templates, you will need to add classes to make the images align.{example below - but be careful as it may change many things on your site}
**fourth** - alt txt **larger than one word** needs to be in speech marks
**fifth** -  re:  [alt text](https://github.com/datenstrom/yellow/discussions/480#discussioncomment-243369 #)

css from standard template
```
.content .left {
    float: left;
    margin: 0 1em 0 0;
}
.content .center {
    display: block;
    margin: 0 auto;
}
.content .right {
    float: right;
    margin: 0 0 0 1em;
}
```

**Reply by uranomaloka** [reply]
I understood it wrong. I, for example, thought there had to be a "-" before the size tag.
That makes it all understandable. 
Thank you @nzyeehaa !
